manuskript/ui/views/textEditView.py

```diff
@@ -264,9 +260,9 @@ def update(self, topLeft, bottomRight):
 
         if self._updateIndexFromID:
             # We have to update to a new index
-            self._index = self._index.model().getIndexByID(
-                self._updateIndexFromID,
-                self._column)
+            index = self._index.model().getIndexByID(self._updateIndexFromID,
+                                                     self._column)
```

Comment:
Do we still need this? `_updateIndexFromID` is not used anymore.

Reply:
You are right. We do not need this.